packages/next/taskfile.js

```diff
@@ -758,28 +758,6 @@ export async function ncc_mini_css_extract_plugin(task, opts) {
     .target('compiled/mini-css-extract-plugin')
 }
 
-// eslint-disable-next-line camelcase
-export async function ncc_webpack_bundle4(task, opts) {
-  const bundleExternals = {
-    ...externals,
-    'schema-utils': externals['schema-utils2'],
-    'webpack-sources': externals['webpack-sources1'],
```

Comment:
Can we drop `schema-utils2` and `webpack-sources1` too or is this still used by something else?